src/Shell/Task/ModelTask.php

```diff
@@ -273,9 +267,15 @@ public function findBelongsTo($model, array $associations)
                         $tmpModelName = Inflector::camelize($found);
                     }
                 }
+                $foreignKeys = (array)TableRegistry::get($tmpModelName)->primaryKey();
```

Comment:
Wouldn't this statement throw an exception if the `$schema`contains non-database IDs like a `webservice_id` or something?